src/status_ctrl.js

```diff
@@ -273,6 +301,22 @@ export class StatusPluginCtrl extends MetricsPanelCtrl {
 				this.display.push(series);
 			}
 		}
+
+		series.display_value = this.formatDisplayValue(series.display_value, target);
+	}
+
+	formatDisplayValue(value, target) {
+        if (target.valueHandler == "Number Threshold" && StatusPluginCtrl.isFloat(value)) {
```

Comment:
The `isFloat` call causes exceptions when a `null` value is passed to it. please fix the method, or make sure that you do not pass null